tests/test_estimation.py

```diff
@@ -0,0 +1,62 @@
+import numpy as np
+import pymc3 as pm
+import theano.tensor as tt
+
+from pymc3_hmm.distributions import DiscreteMarkovChain, PoissonZeroProcess
+from pymc3_hmm.step_methods import FFBSStep
+
+
+def simulate_poiszero_hmm(
+    N,
+    observed,
+    mu=10.0,
+    pi_0_a=np.r_[1, 1],
+    p_0_a=np.r_[5, 1],
+    p_1_a=np.r_[1, 1],
+    pi_0=None,
+):
+    p_0_rv = pm.Dirichlet("p_0", p_0_a)
+    p_1_rv = pm.Dirichlet("p_1", p_1_a)
+    P_tt = tt.stack([p_0_rv, p_1_rv])
+    P_rv = pm.Deterministic("P_tt", tt.shape_padleft(P_tt))
+    if pi_0 is not None:
+        pi_0_tt = tt.as_tensor(pi_0)
+    else:
+        pi_0_tt = pm.Dirichlet("pi_0", pi_0_a)
+    S_rv = DiscreteMarkovChain("S_t", P_rv, pi_0_tt, shape=N)
+    S_rv.tag.test_value = (observed > 0) * 1
+    return PoissonZeroProcess("Y_t", mu, S_rv, observed=observed)
+
+
+def test_gamma_0_estimation():
+
+    np.random.seed(2032)
+    true_initial_states = np.array([0.02, 0.98])
+
```

Comment:
seems like with true initial states set to [.02, .98] the initial state estimation fails. the error im seeing is

```python
E       AssertionError: 
E       Arrays are not almost equal to 1 decimals
E       
E       Mismatched elements: 2 / 2 (100%)
E       Max absolute difference: 0.25904421
E       Max relative difference: 12.95221072
E        x: array([0.3, 0.7])
E        y: array([0., 1.])
```